### ISSUE TITLE
 Syntax error: "(" unexpected, when running intelephense from terminal

### ISSUE THREAD

**Issue by predragnikolic** [bmewburn/intelephense-docs#4]
intelephense@1.0.8
Ubuntu 18.10

Trying to run ` intelephense --stdio` directly in terminal, results in an error:

```
λ  ~  intelephense --stdio
/home/predrag/.npm-global/bin/intelephense: 1: /home/predrag/.npm-global/bin/intelephense: /*!: not found
/home/predrag/.npm-global/bin/intelephense: 2: /home/predrag/.npm-global/bin/intelephense: 2019-03-03-130858_817x354_scrot.png: not found
/home/predrag/.npm-global/bin/intelephense: 3: /home/predrag/.npm-global/bin/intelephense: 2019-03-03-130858_817x354_scrot.png: not found
/home/predrag/.npm-global/bin/intelephense: 4: /home/predrag/.npm-global/bin/intelephense: Syntax error: "(" unexpected
```

But running  `λ  ~  node ./.npm-global/bin/intelephense --stdio` works.
I can verify that `intelephense` is available in my $PATH.

If you need any information, please say.

**Comment by bmewburn** [comment]
Looks like I've forgotten the `#!/usr/bin/env node` at the top of the file. I'll put it in next release.

**Comment by predragnikolic** [comment]
BTW, really great work you've done, thank you!

**Comment by astenman** [comment]
I also had to edit the "intelephense.cmd" file in order to get it running on windows (borrowed code from another langserver ...)

```dos
@IF EXIST "%~dp0\node.exe" (
  "%~dp0\node.exe"  "%~dp0\node_modules\intelephense\lib\intelephense.js" %*
) ELSE (
  @SETLOCAL
  @SET PATHEXT=%PATHEXT:;.JS;=;%
  node  "%~dp0\node_modules\intelephense\lib\intelephense.js" %*
)
```

**Comment by bmewburn** [comment]
resolved in 1.0.10
